src/lib.rs

```diff
@@ -298,7 +298,9 @@
 //! #[macro_use(await)]
 //! extern crate nb;
```

Comment:
Probably should modernize this, too, while we are at it?

Reply:
With Rust 2018 it does not build due to await being a reserved keyword.